mixlr-repeater.gemspec

```diff
@@ -1,6 +1,8 @@
+require_relative 'lib/version'
```

Comment:
Should the gem `lib` directory be added to the `LOAD_PATH` and rely on `require` instead of `require_relative`?